### DISCUSSION TITLE
Saving Sessions

### DISCUSSION THREAD

**Discussion by Dynamic5912** [BottlecapDave/HomeAssistant-OctopusEnergy#785] [Q&A]
How do I setup alerts for Saving Sessions?

I have a Binary Sensor and Sensor that appear to be for Saving Sessions, but they have a note next to them stating "**This entity is no longer being provided by the octopus_energy integration. If the entity is no longer in use, delete it in settings.**__"

Is this correct?

Also once setup, can HA automatically opt me into Saving Sessions? Or do I need to manually do that still?

Thanks!

**Comment by SteveGBuck** [comment]
I think those sensors got replaced with sensors that reference them as octoplus saving sessions?

You can auto join saving sessions using the blueprint documented [here](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/blueprints/)

**Reply by Dynamic5912** [reply]
Thanks. I guess that sends a Persistent Notification?

How can I make it give me a push notification on my device?

**Reply by SteveGBuck** [reply]
I think that blueprint just auto joins you to the saving session. If you wanted to get a HA notification when the saving session has started you'd need to create an automation that is triggered when binary_sensor.octopus_energy_X_XXXXXXXX_octoplus_saving_sessions (replace X with your specific entity ID) switches to ON.

Then under action / "then do" use the notification action and customise to your preference.
